### PR TITLE
Some further corrections to the Managing Hosts guide

### PR DESCRIPTION
**Pull request by spetrosi**
Bug 1835683 - [upstream] Move the Managing Hosts upstream

https://bugzilla.redhat.com/show_bug.cgi?id=1835683

### CODE REVIEW

File: guides/upstreamize.rb [hunk 1]
```diff
@@ -42,7 +42,11 @@ def putsv(str)
             line.gsub!(/Capsule( |\u{00A0}|{nbsp})Server/, '{SmartProxyServer}')
             line.gsub!(/Capsules/, '{SmartProxies}')
             line.gsub!(/Capsule/, '{SmartProxy}')
-            line.gsub!(/satellite.example.com/, '{foreman-example-com}')
+            line.gsub!(/satellite\.example\.com/, '{foreman-example-com}')
+            line.gsub!(/capsule\.example\.com/, '{smartproxy-example-com}')
```
Comment (spetrosi) [inline guides/upstreamize.rb:46]: @lzap, I need your help with those two lines 45, 46. The way it is now replacement does not work. Escaping dots with backslashes did not help too. Do you know how to tell it to replace satellite.example.com? Thanks!

Comment (lzap) [reply guides/upstreamize.rb:46]: What you mean by "it does not work" exactly?

```
$ irb
>> line = "Test for satellite.example.com"
=> "Test for satellite.example.com"
>> line.gsub!(/satellite\.example\.com/, '{foreman-example-com}')
=> "Test for {foreman-example-com}"
```

**Comment by lzap** [comment]
This error looks irrelevant:

```
URL        `https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html/installing_satellite_server_from_a_connected_network/performing_additional_configuration_on_satellite_server#using-external-databases_satellite'

Name       `Using External Databases with Foreman'

Parent URL file:///home/travis/build/theforeman/foreman-documentation/guides/build/doc-Installing_Satellite_Server_Disconnected/index-foreman-deb.html, line 860, col 400

Real URL   https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html/installing_satellite_server_from_a_connected_network/performing_additional_configuration_on_satellite_server#using-external-databases_satellite

Check time 2.889 seconds

Result     Error: 404 Not Found
```

File: guides/upstreamize.rb (hunk 1 above)
Comment (spetrosi) [reply guides/upstreamize.rb:46]: When I run the script it does not pick satellite.example.com, does not replace it. And escaping with backslashed did not help. I see  structures like `|\u{00A0}` above, maybe you know how to escape it properly.

**Pull request merged**